reposerver/swagger.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
Why do we need swagger for reposerver ? Repository server is internal and implements only GRPC. Please delete if we don't need it

Reply:
@alexmt this is a good question.  The command tweak in `generate-proto.sh` generates Swagger files for _all_ ProtoBuf files.  That means that we either need to manually remove the already-created Swagger file for the `reposerver/` component, or simply apply the same logic we do for the `server/` component.  I erred on the side of more documentation since it seemed less kludgy.  I can simply do a `find reposerver -name '*.swagger.json` -delete` instead if that is preferable.